Short-Answer/Algorithms_Answers.md

```diff
@@ -3,12 +3,13 @@
 ## Exercise I
 
 a) O(n) -- linear (grows in direct proportion of the input size)
-
+O(1) * O(n^3) * O(n^2)
 
 b) O(n log n) -- logarithmic 
+O(1) * O(n) * O(1) * O(n) * O(1) * O(1) = O(n log n)
 
```

Comment:
you dont want to multiply in every case, you multiply nested loops in a very generic sense